app/app.iml

```diff
@@ -73,42 +73,51 @@
       <sourceFolder url="file://$MODULE_DIR$/src/test/assets" type="java-test-resource" />
```

Comment:
This should do the trick. Next time when committing make sure not to add it. http://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-git-repo
If you face a problem, a quick google search should help. There are plenty of related questions in Stack Overflow. Once you add the commit, try to squash your commits so it should appear as one. If not at least remove the app.iml fie then I will squash your commits when merging.